libs/coin-modules/coin-aptos/src/api/index.ts

```diff
@@ -22,7 +22,7 @@ export function createApi(
       craftTransaction(client, transactionIntent),
     estimateFees: (transactionIntent: TransactionIntent<AptosAsset, AptosExtra, AptosSender>) =>
       client.estimateFees(transactionIntent),
-    getBalance: (address): Promise<Balance<AptosAsset>[]> => getBalance(client, address),
+    getBalance: (address): Promise<Balance<AptosAsset>[]> => getBalances(client, address),
```

Comment:
`getBalance: (address: string) => getBalances(client, address)`

As my comment above, it is confusing to have `getBalance` call `getBalances`, i would prefer keep it `getBalance`. ADYT?

Reply:
It was created with a different name, the getBalances (plural), because AptosAPI already had a getBalance with a different return (number), and the getBalances returns an array of AptosBalances (which contains the contractAddress and the balance itself). After the merge of the Aptos Token support, both methods became similar, and getBalances were kept.

IMO, `getBalances` it's better semantically because the method returns an array of items, but we can change it...